playkit/src/main/java/com/kaltura/playkit/PlayerDecoratorBase.java

```diff
@@ -37,6 +39,11 @@ public void prepare(@NonNull PKMediaConfig mediaConfig) {
         player.prepare(mediaConfig);
     }
 
+    @Override
+    public void setAdvertising(@NonNull AdvertisingConfig advertisingConfig, @NonNull PKAdvertisingController pkAdvertisingController) {
```

Comment:
i think switching the order of params required first controller than config?

Reply:
Done